API_STYLE_GUIDE.md

```diff
@@ -1,9 +1,11 @@
-# API reference page template
+# API Style Guide
```

Comment:
Vale catch.

```suggestion
# API style guide
```

Reply:
Yeah, in this case I think Vale is wrong :)